src/pages/namespace/Explorer/Workflow/Overview/Services.tsx

```diff
@@ -13,7 +13,7 @@ import { useWorkflowServices } from "~/api/services/query/services";
 const Services = ({ workflow }: { workflow: string }) => {
   const { t } = useTranslation();
   const [dialogOpen, setDialogOpen] = useState(false);
-  const [rebuildService, setRebuildService] = useState<ServiceSchemaType>();
+  const [serviceToRebuild, setServiceToRebuild] = useState<ServiceSchemaType>();
```

Comment:
When the variable is used further below, it is more obvious than "rebuildService", which at first sight sounds like an actiom/method rather than an object. But I don't insist on changing this. We have similar naming patterns elsewhere (e.g. `deleteRegistry`) so I should probably accept that these variable names are not always 100% obvious.